### PR TITLE
fix: add setImplicitWait zero to calculate the coordinate

### PR DESCRIPTION
**Pull request by KazuCocoa**
I'll test this more later, but this should help https://github.com/appium/appium/issues/14988

### CODE REVIEW

File: lib/commands/web.js [hunk 1]
```diff
@@ -230,7 +230,14 @@ extensions.translateWebCoords = async function translateWebCoords (coords) {
   const cmd = '(function () { return {w: window.innerWidth, h: window.innerHeight}; })()';
   const wvDims = await this.remote.execute(cmd);
 
-  await this.getExtraTranslateWebCoordsOffset(wvPos, realDims);
+  // keep track of implicit wait, and set locally to 0
+  const implicitWaitMs = this.implicitWaitMs;
+  try {
+    this.setImplicitWait(0);
```
Comment (mykola-mokhnach) [inline lib/commands/web.js:236]: is this a synchronous call?

**Comment by KazuCocoa** [comment]
it seems this has no negative impact by this since `getExtraTranslateWebCoordsOffset` only has find_element to check some banner thing like app link

**Comment by KazuCocoa** [comment]
I confirmed this helps `setImplicitWait`

File: lib/commands/web.js [hunk 2]
```diff
@@ -230,7 +230,14 @@ extensions.translateWebCoords = async function translateWebCoords (coords) {
   const cmd = '(function () { return {w: window.innerWidth, h: window.innerHeight}; })()';
   const wvDims = await this.remote.execute(cmd);
 
-  await this.getExtraTranslateWebCoordsOffset(wvPos, realDims);
+  // keep track of implicit wait, and set locally to 0
+  const implicitWaitMs = this.implicitWaitMs;
+  try {
+    await this.setImplicitWait(0);
```
Comment (mykola-mokhnach) [inline lib/commands/web.js:236]: this call could be outside of the `try` block

File: lib/commands/web.js [hunk 3]
```diff
@@ -230,7 +230,14 @@ extensions.translateWebCoords = async function translateWebCoords (coords) {
   const cmd = '(function () { return {w: window.innerWidth, h: window.innerHeight}; })()';
   const wvDims = await this.remote.execute(cmd);
 
-  await this.getExtraTranslateWebCoordsOffset(wvPos, realDims);
+  // keep track of implicit wait, and set locally to 0
```
Comment (mykola-mokhnach) [inline lib/commands/web.js:233]: I would rather add a link to the original issue into comments

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**